### DISCUSSION TITLE
Missing Documentation Link

### DISCUSSION THREAD

**Discussion by HarshithaBR** [empriselab/RCareWorld#102] [Help - Using] [answered]
Hi Team

The documentation link for the challenge : https://emprise.cs.cornell.edu/rcareworld-doc/ seems to be broken
can you please direct us to the right link?

Thanks and best regards
Harshitha

**Comment by YoruCathy** [comment, marked as answer]
Hi @HarshithaBR this link is deprecated. Please refer to the link in the readme in the latest commit: https://rcareworld.readthedocs.io/en/phy-robo-care/

**Reply by HarshithaBR** [reply]
Thank you!
We will refer to the updated link.
